views/agenda.tmpl

```diff
@@ -0,0 +1,121 @@
+                    <h4 class="mb-2">{{.Id}}
+                    </h4>
+                </div>
+            </div>
+            <div class="row justify-content-between">
+                <div class="col-md-7 col-sm-6 d-flex">
+                    <table class="">
+                        <tr>
+                            <td class="text-right pr-2 lh1rem vam nowrap xs-w117">Description</td>
+                            <td>
+                                <span class="hash break-word  lh1rem">{{.Description}}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="text-right pr-2 lh1rem vam nowrap xs-w117">StartTime</td>
+                            <td>
+                                <span class="hash break-word  lh1rem">{{TimeConversion .StartTime}}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="text-right pr-2 lh1rem vam nowrap xs-w117">ExpireTime</td>
+                            <td>
+                                <span class="hash break-word  lh1rem">{{TimeConversion .ExpireTime}}</span>
+                            </td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="col-md-5 col-sm-6 d-flex">
+                    <table class="">
+                        <tr>
+                            <td class="text-right lh1rem pr-2 xs-w117">Status</td>
+                            <td class="lh1rem" style="min-width: 178px;">
+                                <span class="hash break-word  lh1rem">{{.Status}}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="text-right lh1rem pr-2 xs-w117">QuorumProgress</td>
+                            <td class="lh1rem" style="min-width: 178px;">
+                                <span class="hash break-word  lh1rem">{{.QuorumProgress}}</span>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="text-right lh1rem pr-2 xs-w117">VoteVersion</td>
+                            <td class="lh1rem" style="min-width: 178px;">
+                                <span class="hash break-word  lh1rem">{{.VoteVersion}}</span>
+                            </td>
+                             <tr>
+                            <td class="text-right pr-2 lh1rem vam nowrap xs-w117">Mask</td>
+                            <td>
+                                <span class="hash break-word  lh1rem">{{.Mask}}</span>
+                            </td>
+                        </tr>
+                        </tr>
+                    </table>
+                </div>
+            </div>
+            {{if .Choices}}
+            <table class="table table-mono-cells table-sm striped">
```

Comment:
Tested it on mobile safari and the performance is good. 
But there is a bit small usability issue due to this table pushing out past the defined page container.
This causes the page to be sideways scrollable, which interferes with touch gestures when trying to pan and zoom the chart range slider.

But if we add the 'table-responsive' bootstrap class to this table, the table itself becomes sideways scrollable, instead of the whole page and solves this problem.